libs/core/src/lib/dialog/dialog.component.ts

```diff
@@ -24,6 +25,7 @@ import { DialogRef } from './utils/dialog-ref.class';
 import { applyCssClass } from '../utils/decorators/apply-css-class.decorator';
 import { CssClassBuilder } from '../utils/interfaces/css-class-builder.interface';
 import { DialogBase } from './base/dialog-base.class';
+import { RtlService } from '../utils/public_api';
```

Comment:
please prefer reference to the direct entity instead of a barrel import, in some cases it causes build issues